account_authentications.go

```diff
@@ -39,6 +39,66 @@ type (
 		DeleteTime                    *time.Time        `json:"delete_time,omitempty"`
 	}
 
+	// AccountAuthenticationMethodUpdateRequest request DTO for AccountAuthenticationMethodUpdate
+	// https://api.aiven.io/doc/#operation/AccountAuthenticationMethodUpdate
+	AccountAuthenticationMethodUpdateRequest struct {
+		Enabled                bool              `json:"authentication_method_enabled,omitempty"`
+		Name                   string            `json:"authentication_method_name"`
+		AutoJoinTeamId         string            `json:"auto_join_team_id,omitempty"`
+		SAMLCertificate        string            `json:"saml_certificate,omitempty"`
+		SAMLDigestAlgorithm    string            `json:"saml_digest_algorithm,omitempty"`
+		SAMLEntity             string            `json:"saml_entity_id,omitempty"`
+		SAMLFieldMapping       *SAMLFieldMapping `json:"saml_field_mapping,omitempty"`
+		SAMLIdpLoginAllowed    bool              `json:"saml_idp_login_allowed,omitempty"`
+		SAMLIdpUrl             string            `json:"saml_idp_url,omitempty"`
+		SAMLSignatureAlgorithm string            `json:"saml_signature_algorithm,omitempty"`
+		SAMLVariant            string            `json:"saml_variant,omitempty"`
+	}
+
+	// AccountAuthenticationMethodReplaceRequest sends all fields explicitly
+	AccountAuthenticationMethodReplaceRequest struct {
+		Enabled                bool              `json:"authentication_method_enabled"`
+		Name                   string            `json:"authentication_method_name"`
+		AutoJoinTeamId         string            `json:"auto_join_team_id"`
+		SAMLCertificate        string            `json:"saml_certificate"`
+		SAMLDigestAlgorithm    string            `json:"saml_digest_algorithm"`
+		SAMLEntity             string            `json:"saml_entity_id"`
+		SAMLFieldMapping       *SAMLFieldMapping `json:"saml_field_mapping"`
+		SAMLIdpLoginAllowed    bool              `json:"saml_idp_login_allowed"`
+		SAMLIdpUrl             string            `json:"saml_idp_url"`
+		SAMLSignatureAlgorithm string            `json:"saml_signature_algorithm"`
+		SAMLVariant            string            `json:"saml_variant"`
+	}
+
+	// AccountAuthenticationMethodUpdated response DTO for AccountAuthenticationMethodUpdate
```

Comment:
What is the `DTO` abbreviation? Can you replace it in the comments with something else everyone can easily understand? Perhaps all abbreviations except API are not that welcome in the annotations.

Reply:
DTO is data transfer object. It's widely used word https://martinfowler.com/eaaCatalog/dataTransferObject.html

Reply:
Thanks for the explanation. Would you consider rephrasing comments with something else?